### PR TITLE
Workshop mailers adjusted for new surveys

### PR DESCRIPTION
**Pull request by mehalshah**

### CODE REVIEW

File: dashboard/app/mailers/pd/workshop_mailer.rb [hunk 1]
```diff
@@ -229,11 +230,7 @@ def get_details_partial(course, subject)
     nil
   end
 
-  def teacher_enrollment_subject(workshop, use_pre_survey_subject: false)
-    if use_pre_survey_subject && workshop.pre_survey?
```
Comment (mehalshah) [inline dashboard/app/mailers/pd/workshop_mailer.rb:233]: We don't want this old subject anymore

File: dashboard/app/views/pd/workshop_mailer/_csd_pre_course_reading.html.haml [hunk 2]
```diff
@@ -0,0 +1,28 @@
+%h4
```
Comment (mehalshah) [inline dashboard/app/views/pd/workshop_mailer/_csd_pre_course_reading.html.haml:1]: I was hoping to make these two one file, but they are just too different on link content. Two files seemed cleaner

File: dashboard/app/views/pd/workshop_mailer/_csd_supply_list.html.haml [hunk 3]
```diff
@@ -1,12 +0,0 @@
-%h3
```
Comment (mehalshah) [inline dashboard/app/views/pd/workshop_mailer/_csd_supply_list.html.haml]: The supply list on the other hand can be fully removed

File: dashboard/app/views/pd/workshop_mailer/_teacher_enrollment_details.html.haml [hunk 4]
```diff
@@ -8,50 +8,14 @@
     vertical-align: top;
   }
 
-%p
```
Comment (mehalshah) [inline dashboard/app/views/pd/workshop_mailer/_teacher_enrollment_details.html.haml:11]: Moved into the pre course survey partial

File: dashboard/app/views/pd/workshop_mailer/_teacher_enrollment_details.html.haml [hunk 5]
```diff
@@ -8,50 +8,14 @@
...
-  We’re excited to continue to work together to bring computer science to your classroom.
-  - if @workshop.local_summer? && @is_reminder
-    Please take a moment to complete the pre-course survey and review the
-    following logistical details about your workshop.
-  If you have any questions about your workshop, reach out to your workshop organizer directly:
-  = @workshop.organizer.name
-  at
```
Comment (aoby) [inline dashboard/app/views/pd/workshop_mailer/_teacher_enrollment_details.html.haml:18]: It looks like some of this stuff should remain. This line and line 12 are common, outside the local_summer conditional.

**Comment by tanyaparker** [comment]
[Update reminder emails for local summer workshops](https://trello.com/c/ujwaQJTy/609-update-reminder-emails-for-local-summer-workshops)

**Pull request merged**